Basic-Car-Maintenance/Documentation.docc/Documentation.md

```diff
@@ -16,3 +16,5 @@ Icons have the following colors:
 * Blue #253AA7
 * Black #000000
 * Orange #F78616
+
+[Firestore Details](FirestoreDetails.md)
```

Comment:
```suggestion
<doc:FirestoreDetails>
```

This still didn't work for me

This was the proper syntax, found from [this example](https://developer.apple.com/documentation/xcode/slothcreator_building_docc_documentation_in_xcode)